app/assets/javascripts/discourse/components/bulk-select-button.js

```diff
@@ -4,6 +4,18 @@ import showModal from "discourse/lib/show-modal";
 export default Component.extend({
   classNames: ["bulk-select-container"],
 
+  didInsertElement() {
+    this._super(...arguments);
+
+    let mainOutletPadding =
```

Comment:
I think should be in an afterRender block

Reply:
Yes should be good, your only risk now is that the node is actually not available because some fast transition removed the component before it rendered but still called the block. That could show an error but wouldn't cause any crash, so no need to worry about it probably.